views/default/admin/upgrades.php

```diff
@@ -1,29 +1,13 @@
-<?php
+-<?php
```

Comment:
what happened here??

Reply:
Meh. Don't know. Fingers faster than my brain.

Reply:
> Fingers faster than my brain

that can be very dangerous when programming, maybe not in other parts of your life though ;)

Reply:
could do a manual test from 2.3?